migration/create-migration-document.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -euo pipefail
+
+usage() {
+    echo "This script must have apps old and new major versions as arguments."
+    echo "Usage: $0 [old_version] [new_version]"
+    echo "Example: ./migration.sh v0.22 v0.23"
+    }
+    if [  $# -lt 2 ]; then
+        usage
+        exit 1
+    fi
+
+here="$(dirname "$(readlink -f "$0")")"
+export old_version="${1}"
+export new_version="${2}"
+folder_name="${here}/v${old_version}.x-v${new_version}.x"
```

Comment:
In the usage instructions you say to include the `v` while here you expect them not to. But I would say that you could assume their there (or implement some logic to support both I guess)

```suggestion
folder_name="${here}/${old_version}.x-${new_version}.x"
```